tests/testdata/outputs/ether_send.sol.easm

```diff
@@ -0,0 +1,398 @@
+0 PUSH1 0x60
```

Comment:
Are the testdata outputs generated during every test run? If yes, should we add this directory to .gitignore?

Reply:
No, they should not be put into .gitignore.

The `outputs` actually means expected, they are needed for running testing. The tests will run new code to generate outputs in memory and compare to the files inside `outputs` to see if there is any changes. 

If there is any changes, developers can view it and decide if it's OK. If the changes is expected, developers can uncommented code in test cases(you can find it in several test cases like `report_tests.py`) to generate new outputs and committed them as new expected outputs.

Reply:
I see, thanks.